src/v2/guide/plugins.md

```diff
@@ -37,7 +37,7 @@ Vue.use(MyPlugin, { someOption: true })
 
 `Vue.use` automatically prevents you from using the same plugin more than once, so calling it multiple times on the same plugin will install the plugin only once.
 
-Some plugins provided by Vue.js official plugins such as `vue-router` automatically calls `Vue.use()` if `Vue` is available as a global variable. However in a module environment such as CommonJS, you always need to call `Vue.use()` explicitly:
+Some official plugins provided by Vue.js such as `vue-router` automatically calls `Vue.use()` if `Vue` is available as a global variable. However in a module environment such as CommonJS, you always need to call `Vue.use()` explicitly:
```

Comment:
```suggestion
Some official plugins provided by Vue.js such as `vue-router` automatically call `Vue.use()` if `Vue` is available as a global variable. However in a module environment such as CommonJS, you always need to call `Vue.use()` explicitly:
```

Reply:
We can even remove "provided by Vue.js" because that's literally what "official" means.